STANDARD.md

```diff
@@ -100,6 +100,16 @@ Each file's *root* directory (path `/`) must at leat contain the attributes:
                    of `%T` with the integer value of the iteration, e.g.,
                    `/data/%T` becomes `/data/100`
     - allowed value: fixed to `/data/%T/` for this version of the standard
+	- remark: all the data that is formatted according to the present
+      standard (i.e. both the meshes and the particles) is to be
+      stored within a path of the form given by `basePath` (e.g. in
+      the above example, the data will be stored within the path `/data/100/`).
+      If, for various reasons, a user wants to store **additional
```

Comment:
formatting-wise: I think we currently *stress* important words only with a simple italic `*stressed fact*` in the text.